index.yml

```diff
@@ -318,7 +318,7 @@
 - url: https://github.com/cloudfoundry-community/sawmill-boshrelease
 - url: https://github.com/SAP/ipsec-release
   min_version: 3
-- url: https://github.com/bosh-prometheus/prometheus-boshrelease
+- url: https://github.com/cloudfoundry/prometheus-boshrelease
 - url: https://github.com/bosh-prometheus/node-exporter-boshrelease
```

Comment:
This release has also moved and is subject to being updated here either.
See: https://github.com/cloudfoundry/community/blob/a81540a/toc/working-groups/foundational-infrastructure.md?plain=1#L385-L389

Reply:
Hello @bgandon, cloudfoundry/bosh_exporter, cloudfoundry/cf_export and cloudfoundry/firehose_exporter are not in https://github.com/cloudfoundry/bosh-io-releases. I have modified the path of[ node-exporter-boshrelease](https://github.com/cloudfoundry/node-exporter-boshrelease)

Reply:
Hello [@bgandon](https://github.com/bgandon) , **cloudfoundry/bosh_exporter**, **cloudfoundry/cf_exporter** and **cloudfoundry/firehose_exporter** are not in https://github.com/cloudfoundry/bosh-io-releases. I also modified the path of [node-exporter-boshrelease](https://github.com/cloudfoundry/node-exporter-boshrelease) in this PR